doc/arch/002-bottom-sheets.md

```diff
@@ -0,0 +1,22 @@
+# ADR: Using Activity for bottom sheets instead of Fragment
+
+## Status
+Accepted
+
+## Context
+
+For displaying non-persistent bottom sheets, Material.io recommends using BottomSheetDialogFragment. Unfortunately it doesn’t play very nicely with the soft keyboard when the sheet includes input text fields. When the soft keyboard is shown, it aligns itself with the bottom edge of the field. Everything beneath the focused text field gets covered by the keyboard. 
+
+![BottomSheetDialogFragment without a focused text field](https://raw.githubusercontent.com/simpledotorg/simple-android/master/doc/arch/images/bottomsheetdialogfragment.png)
+
+![BottomSheetDialogFragment with a focused text field](https://raw.githubusercontent.com/simpledotorg/simple-android/master/doc/arch/images/bottomsheetdialogfragment_with_keyboard.png)
+
+This is a standard behavior of the soft keyboard, but Activities have worked around this by panning or resizing the layout whenever the keyboard is shown. For some reason, changing the softInputMode on the dialog doesn’t have any affect on dialogs.
```

Comment:
"this is a standard" -> "this is standard"

Reply:
There are other references to "soft keyboard" that are still present in the file. Please update.

Reply:
Whoops, sorry for that.